File: Matlab/SCRoF_Fevrier/asPaireAvec.m
function rP=asPaireAvec(AS,v)
% rP=asPaireAvec(AS,v);
% Retourne les rangs de AS.Cpaires qui impliquent la variable v
P=AS.Cpaires;
rP=find((P(:,1)==v) + (P(:,2)==v));

File: R/asPaireAvec.R
asPaireAvec <- function(AS, v) {
  # Retourne les rangs de AS$Cpaires qui impliquent la variable v
  
  P <- AS$Cpaires
  rP <- which(P[1, ] == v | P[2, ] == v)
  
  return(rP)
}